Xamarin.Forms.Platform.WinRT/FormsComboBox.cs

```diff
@@ -49,6 +54,12 @@ protected override void OnApplyTemplate()
 					IsClosingAnimated = true;
 				}
 			}
+			Popup = (Popup)GetTemplateChild("Popup");
+			if (Popup != null)
+			{
+				Popup.Opened += (s, e) => IsPopupOpen = true;
```

Comment:
I wonder if we need to unsubscribe these somewhere, too.

Reply:
I made that first change mentioned above, but I'm not certain where where we would need to unsubscribe these, if we do.